src/modules/login/containers/LoginPage.js

```diff
@@ -0,0 +1,36 @@
+import { h, Component } from "preact";
+import { Page } from "src/components/Page";
+import I18n from "src/config/i18n";
+import GoogleLogin from "react-google-login";
+import { GOOGLE_CLIENT_ID, GOOGLE_HOSTED_DOMAIN } from "src/globals";
+
+import styles from "./LoginPage.scss";
+
+const responseGoogle = () => {};
+
+export class LoginPage extends Component {
+  render() {
+    return (
+      <Page>
+        <main class={styles.main}>
+          <div class={styles.logo} />
+          <div class={styles.login}>
+            <GoogleLogin
```

Comment:
How are you going to implement authentication to the kudo backend, if you do Google auth directly in the frontend?

Reply:
Still trying to get our mind around this; Should authenticate at backend, using google credentials.

Reply:
Should be able to send the ID token to the backend, validating it there using the Google API Client Library.

Reply:
shouldn't we use the resource owner password credentials flow here? just like buurkracht mobile? probably does not answer the question how the backend is going to authenticate the user

Reply:
"In the coming months, we will no longer allow OAuth requests to Google in embedded browsers known as “web-views”,"

https://developers.googleblog.com/2016/08/modernizing-oauth-interactions-in-native-apps.html

Reply:
But using the native device browser should still be possible? Only question is how to redirect back to the app. Asking for full credentials and pass them on would be more convenient, I agree.